packages/react-core/src/components/Select/Select.tsx

```diff
@@ -522,10 +537,12 @@ class Select extends React.Component<SelectProps & InjectedOuiaProps, SelectStat
                   {toggleIcon && <span className={css(styles.selectToggleIcon)}>{toggleIcon}</span>}
                   <span className={css(styles.selectToggleText)}>{placeholderText}</span>
                   {!isCheckboxSelectionBadgeHidden &&
-                    selections &&
-                    (Array.isArray(selections) && selections.length > 0) && (
+                    (customBadgeText !== null ||
```

Comment:
What happens with an empty string or negative numbers? Perhaps we can add some minimal validation for that?

Reply:
A consumer could always pass the string "-1". We're just rendering a passed prop, so I don't think we need to add validation.

Reply:
I agree, I don't think we need to add validation here.